interpreter/interpreter_test.go

```diff
@@ -883,8 +883,8 @@ func TestStack(t *testing.T) {
 			FunctionName: "window",
 			Location: ast.SourceLocation{
 				File:   "universe/universe.flux",
-				Start:  ast.Position{Line: 3814, Column: 12},
-				End:    ast.Position{Line: 3814, Column: 51},
+				Start:  ast.Position{Line: 4031, Column: 12},
+				End:    ast.Position{Line: 4031, Column: 51},
```

Comment:
I don't see any changes to the stdlib other than removing the `experimental/universe` package, so why would this test need to change?

Reply:
Forgot to revert the line after splitting out https://github.com/influxdata/flux/pull/4976 . Fixed.